rpc.go

```diff
@@ -94,12 +93,12 @@ func (r rpc) Channel(service string) transport.Channel {
 		// we can eventually write an outbound that load balances between
 		// known outbounds for a service.
 		return transport.Channel{
-			Outbound: out,
+			Outbound: request.ValidatorOutbound{out},
```

Comment:
I wonder if we should apply this from within the channel, so there's no way to opt-out when constructing a channel by hand.